### ISSUE TITLE
Vulcanized drawer-panel not displayed successfully in IE11

### ISSUE THREAD

**Issue by JustinCase1089** [Polymer/vulcanize#241]
Hi

When vulcanizing this

<!doctype html>
<html>
    <head>
        <script src='bower_components/webcomponentsjs/webcomponents.min.js'></script>
        <link rel="import" href="bower_components/polymer/polymer.html"/>
        <link rel='import' href='bower_components/paper-drawer-panel/paper-drawer-panel.html'>
    </head>
    <body>
    <paper-drawer-panel id="paperDrawerPanel">
        <div drawer>
            drawer
        </div>
        <div main>
            main
        </div>
    </paper-drawer-panel>
    </body>
</html>

It does display correctly in IE. This is the same issue as in https://github.com/Polymer/vulcanize/issues/209, but it seems to have reappeared

Cheers

**Comment by spirylics** [comment]
+1

**Comment by JustinCase1089** [comment]
Please look into this soon. I have a broken production site atm :-(. Thanks

**Comment by Anthony2539** [comment]
+1

**Comment by JustinCase1089** [comment]
@azakus, is there anything I can do to help getting this issue resolved? :-)

**Comment by azakus** [comment]
Sorry for the delay on this.
@JustinCase1089 can you test with vulcanize 1.13.0? I can't seem to reproduce with that version on IE 11.

**Comment by JustinCase1089** [comment]
Hi Daniel, thanks for looking into this :-). 
I ensured that I am using Vulcanize 1.13.0, Polymer 1.1.4 and paper-drawer-panel 1.0.3
I still got the issue with the panel
http://stlucia.dhigroup.com/indexV.html holds the vulcanized page that has the problem in IE, check it out
http://stlucia.dhigroup.com/index.html is the non vulcanized page
http://stlucia.dhigroup.com/index.zip is the zipped site in case you want to have a look.
Let me know if you need more info
Cheers

**Comment by azakus** [comment]
Ah I see the issue now. Using `webcomponents.js` instead of `webcomponents-lite.js` was hiding a bug in `iron-media-query`.
If you don't inline scripts, when `iron-media-query` boots up, `query` is undefined on this line: https://github.com/PolymerElements/iron-media-query/blob/master/iron-media-query.html#L63

**Comment by azakus** [comment]
Hmm, the media-query was only part of the problem.
You're right that the dom-module is not booting up without `--inline-scripts`. Very odd.

**Comment by JustinCase1089** [comment]
Really appreciating you looking into this ;-)

**Comment by azakus** [comment]
It looks like https://github.com/Polymer/polymer/pull/2201 didn't fix this, so it this is a different bug than #209

**Comment by JustinCase1089** [comment]
Are there any workarounds for this issue for the time being?

**Comment by azakus** [comment]
--inline-scripts and --inline-css flags were working for me

**Comment by JustinCase1089** [comment]
Well, I have tried that, but that had a rather negative effect on the leaflet map I am using in the site where I have the original problem. I have placed a version of the site with the inline script + vulcanize here http://greve.dhigroup.com/indexV.html. The site that has been vulcanized without inline script is here http://greve.dhigroup.com ... the issue with the latter is then of couse that it doesn't work in IE. 

The tiles are shuffled very nicely like in a game and you almost feel compelled to begin swapping the tiles around :-)

This may be a separate issue with inlining script. If you would like, I can try and condense this issue down and create a new issue for that

**Comment by JustinCase1089** [comment]
Sorry, ignore above ... it may be me doing something wrong here

**Comment by JustinCase1089** [comment]
Yes, I the workaround works in the site I had the initial problem in. Thanks
